### DISCUSSION TITLE
Whisper skips over LibriVox intros

### DISCUSSION THREAD

**Discussion by ryanheise** [openai/whisper#2127] [General]
LibriVox recordings usually begin with a stock introduction such as the following one for English:

"This is a LibriVox recording. All LibriVox recordings are in the public domain. For more information or to volunteer, visit librivox.org."

It seems that when asked to transcribe a LibriVox recording, Whisper will tend to skip this stock intro and transcribe only the part for which the transcript was available in the LibriVox metadata.

We could make some guesses about what the training data might look like to cause this, although here I just wanted to note the observation.

**Comment by lordgrenville** [comment]
This is wild. Thanks for making this post, as I'm having the same issue and thought I was crazy.
